File: openbis_api/source/java/ch/ethz/sis/openbis/generic/asapi/v3/dto/dataset/id/IContentCopyId.java
package ch.ethz.sis.openbis.generic.asapi.v3.dto.dataset.id;

import ch.ethz.sis.openbis.generic.asapi.v3.dto.common.id.IObjectId;
import ch.systemsx.cisd.base.annotation.JsonObject;
import ch.systemsx.cisd.common.annotation.TechPreview;

/**
 * Holds information that uniquely identifies a content copy in openBIS.
 * 
 * @author anttil
 */
@JsonObject("as.dto.dataset.id.IContentCopyId")
@TechPreview
public interface IContentCopyId extends IObjectId
{
}

File: openbis/source/java/ch/systemsx/cisd/openbis/public/resources/api/v3/as/dto/dataset/id/IContentCopyId.js
/**
 * Holds information that uniquely identifies a content copy in openBIS.
 * 
 * @author anttil
 */
define([ "stjs", "as/dto/common/id/IObjectId" ], function(stjs, IObjectId) {
	var IContentCopyId = function() {
	};
	stjs.extend(IContentCopyId, null, [ IObjectId ], null, {});
	return IContentCopyId;
})